.github/workflows/node-integrate.yml

```diff
@@ -34,6 +35,9 @@ jobs:
               - 'node/packages/aws-lambda-otel-extension/**'
             awsLambdaSdk:
               - 'node/packages/aws-lambda-sdk/**'
+            devModeExtension:
```

Comment:
Let's maybe name it `testGoDevMode` (name should follow package naming, and here it's good to add info that it's about _test_ of _go_